packages/Statie/templates/statie-website/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "dependencies": {
+        "child_process": "^1.0",
+        "gifplayer": "^0.3.3",
```

Comment:
Is this necessary?

Reply:
Oh, not really. It's just copy-paste leftover from my web. Will remove